dask/bytes/local.py

```diff
@@ -15,51 +15,57 @@
 logger = logging.getLogger(__name__)
 
 
-def read_bytes(fn, delimiter=None, not_zero=False, blocksize=2**27,
-        sample=True, compression=None):
+def read_bytes(path, delimiter=None, not_zero=False, blocksize=2**27,
+        sample=True, compression=None, **kwargs):
```

Comment:
Fixed. It was previously receiving `_protocol`, which is not relevant here anyway, so I popped it in core.read_bytes.